ghost/core/core/server/data/schema/schema.js

```diff
@@ -985,7 +985,9 @@ module.exports = {
         started_at: {type: 'dateTime', nullable: true},
         finished_at: {type: 'dateTime', nullable: true},
         created_at: {type: 'dateTime', nullable: false},
-        updated_at: {type: 'dateTime', nullable: true}
+        updated_at: {type: 'dateTime', nullable: true},
+        metadata: {type: 'text', maxlength: 1000000000, fieldtype: 'long', nullable: true},
```

Comment:
Do you have a sense of how big we'd expect this field to actually be? I wonder if allowing a maxlength this large is potentially encouraging misuse of this field?

Reply:
I originally had it like other text fields at 2000 chars. I _think_ that ought to be sufficient, but this is why I was holding off on the migration. We could start with 2000 and go from there?

Reply:
Cool, I'll bump back. Basically, since this is a 'JSON' field I had copied what we had elsewhere. Agree it seems to be overkill and it's easier to bump up than down.

Reply:
Adjusted. If I mark this resolved, does it also resolve it for you? I forget how GH handles that.

Reply:
Sorry, just seeing this — I think it resolves for both of us, but not sure! It's still showing unresolved for me FWIW.